source/user-manual/capabilities/file-integrity/fim-fields-rule-mapping.rst

```diff
@@ -0,0 +1,145 @@
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  hard_links          |  hard_links                 |  List of hard links of the file                                 |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  mode                |  mode                       |  Syscheck scan mode                                             |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  perm                |  perm_after,win_perm_after  |  File permissions                                               |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  uid                 |  uid_after                  |  User ID of the process                                         |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  gid                 |  gid_after                  |  Group ID of the process                                        |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  md5                 |  md5_after                  |  md5 hash of the file in the current alert (after changes)      |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  sha1                |  sha1_after                 |  sha1 hash of the file in the current alert (after changes)     |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  uname               |  uname_after                |  User name of the process                                       |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  gname               |  gname_after                |  Group name of the process                                      |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  mtime               |  mtime_after                |  Timestamp of the file changes                                  |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  inode               |  inode_after                |  Inode of the file in the current alert                         |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  sha256              |  sha256_after               |  sha256 hash of the file in the current alert (after changes)   |
```

Comment:
```suggestion
|  sha256              |  sha256_after               |  sha256 hash of the file in the current event (after changes)   |
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/f3e4eedc32cbd94b8cf88e3709d12d722b0e9dee)

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/d5ce0fcadd5abf8bbb40c82c8ccbba1b333c2572)